client/models.py

```diff
@@ -281,7 +284,62 @@ def image_download_url(self):
             return reverse('download_certificate', kwargs={'pk': self.id})
 
 
+class HelpRequestVolunteerManager(models.Manager):
```

Comment:
The name can be improved, it's currently unclear what this manager is from it's name (does it manage help requests? volunteers? help requests in regards to volunteers or the other way around?)

Reply:
What about OptimalVolunteerManager?